### DISCUSSION TITLE
为啥装两个一样的mod然后点击其中一个的删除，会导致两个都被删除，然而还存留一个图标？

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#2884] [提问] [answered]
如题，log等我玩完了再补，先确认一下这个问题有没有必要修复

**Comment by WForst-Breeze** [comment]
如果两个 Mod 在文件夹中命名不同按理讲这种情况不会发生，上传 log 或提供更详细说明

**Reply by iPodtouch6557** [reply]
两个文件命名相同的
对不起我玩完忘了log了

[Log2.txt](https://github.com/Hex-Dragon/PCL2/files/13403118/Log2.txt)

**Reply by WForst-Breeze** [reply]
怎么做到一个文件夹里出现两个名称相同的模组文件的（震惊）

**Reply by WorldHim** [reply]
> 怎么做到一个文件夹里出现两个名称相同的模组文件的（震惊）

子文件夹？

**Reply by iPodtouch6557** [reply]
就是装了两遍，cf途径没更新也不知道咋搞就装了两遍
后来mr发现更新了 就改了

**Reply by MoYuan-CN** [reply]
啊这... 意外吧

**Comment by LTCatt** [comment, marked as answer]
我觉得有两个名字一样的文件这种事就不在我的考虑范围内……
